src/runtime_src/core/tools/xbtracer/src/lib/capture.h

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <mutex>
+#include "experimental/xrt_ext.h"
+#include "experimental/xrt_hw_context.h"
+#include "experimental/xrt_xclbin.h"
+#include "xrt/xrt_bo.h"
+#include "xrt/xrt_device.h"
+#include "xrt/xrt_kernel.h"
```

Comment:
I don't see any of these used within this header file.  Always include-what-you-use and where-you-use.

Reply:
moved to respective internal headers where they are needed.